settings.py

```diff
@@ -23,6 +23,9 @@ class Settings(BaseSettings):
 
     trace_query_descriptions: bool = False
 
+    # in seconds. delays will not last longer than this.
+    maximum_delay_length: Optional[float] = None
```

Comment:
I think it is worth setting this to 60 seconds. That should be plenty for most testing.

Reply:
The maximum delay length will be set in the gitops manifest. Leaving this code as is.